Palace/Book/UI/TPPBookCellDelegate.m

```diff
@@ -500,8 +503,11 @@ - (void)pollAudiobookReadingLocation
      setLocation:[[TPPBookLocation alloc] initWithLocationString:string renderer:@"NYPLAudiobookToolkit"]
      forIdentifier:self.book.identifier];
     
-    // Save updated location on server
-    [TPPAnnotations postListeningPositionForBook:self.book.identifier selectorValue:string];
+    if ([[NSDate date] timeIntervalSinceDate: self.lastServerUpdate] >= ServerUpdateDelay) {
```

Comment:
Does it mean that in worst case scenario the app doesn't save last 15 seconds of progress? I couldn't find if we upload listening position if the user closes the book.

Reply:
I think saving on exit when the user taps the Back button should be enough

Reply:
Resolved here: https://github.com/ThePalaceProject/ios-audiobooktoolkit/pull/35